agency/src/services/event/events.service.ts

```diff
@@ -0,0 +1,26 @@
+// Initializes the `event` service on path `/event`
```

Comment:
Nit.: do you mind adding a small comment on why we decided to use the internal events service to handle the event loop? I know I will forget as soon as I hit send now, it will make it easier to stick to this pattern later o if it is documented.